gulpfile.js

```diff
@@ -16,10 +16,10 @@ gulp.task('lint', function() {
     .pipe(jshint.reporter( 'jshint-stylish' ))
 });
 
-gulp.task('test', ['lint'], function(done) {
-  return gulp.src('test/*.spec.js', {read: false})
+gulp.task('test', gulp.series('lint', function(done) {
+  return gulp.src('test/tags.spec.js', {read: false})
```

Comment:
Was there a particular reason you're only running `tags.spec` instead of all specs?

Reply:
shoot, no. I had to quiet the noise during my bug hunt and this slipped in.